lib/identity_job_log_subscriber.rb

```diff
@@ -2,13 +2,13 @@
 # https://edgeguides.rubyonrails.org/active_support_instrumentation.html#active-job
 # https://github.com/rails/rails/blob/v6.1.3.1/activejob/lib/active_job/log_subscriber.rb
 class IdentityJobLogSubscriber < ActiveSupport::LogSubscriber
+
   def enqueue(event)
     job = event.payload[:job]
     ex = event.payload[:exception_object]
-
     json = default_attributes(event, job)
 
-    if ex
+    if ex && should_not_warn?(job, ex)
```

Comment:
What if we named this method something like `should_error?`

Reply:
Updated.